x11-wm/pekwm/pekwm-0.2.1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib desktop
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+S="${WORKDIR}"/${PN}\-${PV}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="contrib debug +jpeg +png themes truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libXrender
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	themes? ( dev-vcs/git )"
```

Comment:
```
 * VDB: detected possibly incorrect RDEPEND (x11-wm/pekwm-0.2.1)
 * media-libs/libpng   | media-libs/libpng:=
 *                     > x11-libs/libX11
 *                     > x11-libs/libXext
 * x11-libs/libXrender <
```
So add libX11 and libXext as deps, and verify whether libXrender is needed or not (e.g. the header can be used)

Reply:
Yes, not needed, it compiles correctly with suggested patch.

Reply:
Change commited 771609d.